stable/nextcloud/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: nextcloud
-version: 1.12.0
+version: 1.12.1
 appVersion: 17.0.0
 description: A file sharing server that puts the control and security of your own data back into your hands.
```

Comment:
Also please prefix the description with "DEPRECATED" (see [Deprecating A Chart](https://github.com/helm/charts/blob/master/PROCESSES.md#deprecating-a-chart))

Reply:
done!